pkg/docker/generate.go

```diff
@@ -139,12 +170,18 @@ RUN apt-get update -q && apt-get install -qy --no-install-recommends \
 	pyenv global $(pyenv install-latest --print "%s")`, py, py), nil
 }
 
-func (g *DockerfileGenerator) installCog() string {
-	cogLibB64 := base64.StdEncoding.EncodeToString(cogLibrary)
+func (g *DockerfileGenerator) installCog() (string, error) {
+	cogFilename := "cog." + ksuid.New().String() + ".py"
+	cogPath := filepath.Join(g.Dir, cogFilename)
+	if err := os.WriteFile(cogPath, cogLibrary, 0644); err != nil {
+		return "", fmt.Errorf("Failed to write cog.py: %w", err)
+	}
+	g.generatedPaths = append(g.generatedPaths, cogPath)
 	return g.sectionLabel(SectionInstallingCog) +
 		fmt.Sprintf(`RUN pip install flask requests redis
 ENV PYTHONPATH=/usr/local/lib/cog
-RUN mkdir -p /usr/local/lib/cog && echo %s | base64 --decode > /usr/local/lib/cog/cog.py`, cogLibB64)
+RUN mkdir -p /usr/local/lib/cog
+COPY %s /usr/local/lib/cog/cog.py`, cogFilename), nil
```

Comment:
Maybe a bit neater to put it somewhere that is already in `PYTHONPATH` like `/usr/lib/python{version}/site-packages`?